docs/src/constraints.md

```diff
@@ -876,6 +876,60 @@ julia> @constraint(model, M * y + q ⟂ y)
 [y[1] + 2 y[2] + 5, 3 y[1] + 4 y[2] + 6, y[1], y[2]] ∈ MathOptInterface.Complements(2)
 ```
 
+## Special Ordered Sets (SOS1 and SOS2)
+
+A Special Ordered Set (SOS) is an ordered set of variables with particular characteristics.
```

Comment:
s/with particular/with the following

Reply:
the "s" refers to what?

Reply:
Ah. I forgot this can be non-obvious: https://unix.stackexchange.com/questions/159367/using-sed-to-find-and-replace